src/fmv.cpp

```diff
@@ -25,18 +25,53 @@ static f32 Percent(f32 max, f32 percent)
     return (max / 100.0f) * percent;
 }
 
-static void RenderSubtitles(Renderer& rend, const char* msg, float x, float y, float w, float h)
+static void RenderSubtitles(Renderer& rend, std::vector<const SubTitle*> msg, float x, float y, float w, float h)
 {
+
+	const char* prev_sub = "BUG";
+	const char* current_sub;
     f32 xpos = 0.0f;
     f32 ypos = y + h;
 
     rend.fillColor(ColourF32{ 0, 0, 0, 1 });
     rend.fontSize(Percent(h, 6.7f));
     rend.textAlign(TEXT_ALIGN_TOP);
+	rend.fontFace("sans");
     
     f32 bounds[4];
-    rend.textBounds(static_cast<int>(xpos), static_cast<int>(ypos), msg, bounds);
 
+	if (msg.size() > 1) {
+		prev_sub = msg[0]->Text().c_str();
+		current_sub = msg[1]->Text().c_str();
+	}
+	else {
+		current_sub = msg[0]->Text().c_str();
+	}
+
+
+	//char* temp;
```

Comment:
might as well delete all of the commented out code until the tag parsing is fully implemented, also TABS